spec/rubocop/cop/style/option_hash_spec.rb

```diff
@@ -69,4 +69,34 @@ def cook(instructions, ingredients = { hot: [], cold: [] })
       RUBY
     end
   end
+
+  context 'when passing options hash to super' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<-RUBY.strip_indent)
+        def allowed(foo, options = {})
+          super
+        end
+      RUBY
+    end
+
+    it 'does not register an offense when lines exist before call to super' do
```

Comment:
"Lines exist" sounds strange to me. Perhaps you meant to say "When there's other code before the call to super"?

Reply:
👍

Reply:
So rubocop's `Metrics/LineLength` cop triggered here when I gave more descriptive wording :/
Decided on "when code exists before call to super"